Add Create Task API quality assurance table (#499)
anitab-org/mentorship-backend: docs/quality-assurance-test-cases.md

| Revoke self the admin role when self is not the only admin | Success |
| Revoking an admin user, when the current user is not an admin | Fail |

## Tasks

### Create
**Service:** POST /mentorship_relation/{relation_id}/task

| Test Case       | Outcome |
| ------------- | ------------- |

> |Create a task if there is a valid relation exist between user and mentor and relation state is accepted | Success |

Reviewer: I meant from a source markdown perspective as well. Look at this line

> |Create a task if there is a valid relation exist between user and mentor and relation state is accepted | Success |

Reviewer 2: ```suggestion
| Create a task for a relation, in the accepted state, between logged user and another user | Success |
```

> | Create a task if there is no relation between user and mentor | Fail |

Reviewer 2: I am not sure if anyone could even test this, can you be more specific? I think what you want to say here is something like:
create a task when logged user is not involved in the relation

> | Create a task if relation state is not accepted | Fail |

Reviewer 2: ```suggestion
| Create a task if relation state is different than accepted | Fail |
```
its more clear like this, I think
